scripts/train.py

```diff
@@ -71,11 +71,13 @@ def create_models(num_classes, weights='imagenet', multi_gpu=0):
     return model, training_model, prediction_model
 
 
-def create_callbacks(model, training_model, prediction_model, validation_generator, dataset_type):
+def create_callbacks(model, training_model, prediction_model, validation_generator, dataset_type, snapshot_path):
     callbacks = []
 
     # save the prediction model
-    checkpoint = keras.callbacks.ModelCheckpoint(os.path.join('snapshots', 'resnet50_{dataset_type}_{{epoch:02d}}.h5'.format(dataset_type=dataset_type)), verbose=1)
+    checkpoint = keras.callbacks.ModelCheckpoint(os.path.join(snapshot_path,
+                                                              'resnet50_{dataset_type}_{{epoch:02d}}.h5'
+                                                              .format(dataset_type=dataset_type)), verbose=1)
```

Comment:
I know this is what PEP8 suggests, but I like the alternative more. Ie. : 
```python
foo(
    bar=1,
    baz=2
)
```

Could you change it to that format? The rest of the code is formatted the same way.